core-java-modules/core-java-9/src/test/java/com/baeldung/java9/inputstream/outputstream/InputStreamToOutputStreamUnitTest.java

```diff
@@ -0,0 +1,85 @@
+package com.baeldung.java9.inputstream.outputstream;
+
+import static org.apache.commons.lang3.RandomStringUtils.randomAlphabetic;
+import static org.junit.Assert.assertEquals;
+
+import java.io.*;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.Test;
+
+import com.google.common.io.ByteStreams;
+
+public class InputStreamToOutputStreamUnitTest {
+
+    // buffer size used for reading and writing
+    private static final int BUFFER_SIZE = 8192;
+
+    /**
+     * Reads all bytes from an input stream and writes them to an output stream.
+     * @param source - input stream to copy data from
+     * @param target - output stream to copy data too
+     */
+    private static void copy(InputStream source, OutputStream target) throws IOException {
```

Comment:
Since there isn't a compelling reason to make this method private or static, let's remove those keywords.